.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "liveServer.settings.port": 5501,
+  "liveServer.settings.port": 5502,
   "livePreview.defaultPreviewPath": "/index.html"
```

Comment:
- [ ] [OPTIONAL] you can add this .vscode file to your .gitingore. usually personal setting files that don't have to do with the project can be added to the .gitignore file.